lib/types/themeLibrary/ThemeLibraryFormatter.js

```diff
@@ -0,0 +1,88 @@
+const log = require("@ui5/logger").getLogger("types:themeLibrary:ThemeLibraryFormatter");
+const path = require("path");
+const AbstractUi5Formatter = require("../AbstractUi5Formatter");
+
+
+class ThemeLibraryFormatter extends AbstractUi5Formatter {
+	/**
+	 * Formats and validates the project
+  	 *
+	 * @returns {Promise}
+	 */
+	async format() {
+		const project = this._project;
+		await this.validate();
+
+		log.verbose("Formatting theme-library project %s...", project.metadata.name);
+		project.resources.pathMappings = {
+			"/resources/": project.resources.configuration.paths.src
+		};
+
+		if (project.resources.configuration.paths.test) {
+			// Directory 'test' is somewhat optional for theme-libraries
+			project.resources.pathMappings["/test-resources/"] = project.resources.configuration.paths.test;
+		} else {
+			log.verbose(`Ignoring 'test' directory for project ${project.metadata.name}.` +
+				"Either no setting was provided or the path not found.");
+		}
+	}
+
+	/**
+	 * Validates the project
+	 *
+	 * @returns {Promise} resolves if successfully validated
+	 * @throws {Error} if validation fails
+	 */
+	validate() {
+		const project = this._project;
+		return Promise.resolve().then(() => {
+			if (!project) {
+				throw new Error("Project is undefined");
+			} else if (project.specVersion === "0.1" || project.specVersion === "1.0") {
```

Comment:
@RandomByte do you think this simple check is okay? With this we rely on the general specVersion check within ui5-project to be executed, otherwise also other specVersions would be allowed.

Reply:
LGTM for the time being